### PR TITLE
feat: add privacy policy (WPB-8846)

### PR DESCRIPTION
**Pull request by alexandreferris**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-8846" title="WPB-8846" target="_blank">TaskWPB-8846</a>  [Android] Add privacy policy and terms of use section to Android  
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [X] contains a reference JIRA issue number like `SQPIT-764`
    - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

There was no `Privacy Policy` / `Terms of Use` linking from the App Settings.

### Causes (Optional)

Not implemented.

### Solutions

- Add a new screen : `About this App` from the Settings screen (at the bottom)
- Add Privacy Policy / Terms of Use / Licenses / Current Version GIT commit / Current App Version / Copyright on About this App screen
- Remove Licenses from Settings screen

#### How to Test

- Open App
- Open Settings Screen (from top-left menu)
- Scroll to the bottom and open `About this App`
- Should contain items mentioned in `Solutions` and:
   - Privacy Policy should open on the web
   - Terms of Use should open on the web
   - Licenses should open another screen
   - Git commit / App version should be able to copy to clipboard

### Attachments (Optional)
Settings Screen
About this App Screen

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [4729](https://10.10.124.134/job/AR-build-pipeline/4729/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.8.0(9671307)(9671307)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.8.0(9671307)(9671307)-staging-compat.apk)

**Review by Garzas — APPROVED** [review]
Looking good 🚀

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -197,6 +199,11 @@
     <string name="app_settings_screen_title">App Settings</string>
     <string name="give_feedback_screen_title">Give Feedback</string>
     <string name="report_bug_screen_title">Report Bug</string>
+    <string name="about_app_screen_title">About This App</string>
+    <string name="settings_privacy_policy_label">Privacy Policy</string>
+    <string name="settings_terms_of_use_label">Terms of Use</string>
+    <string name="label_copyright">Copyright</string>
+    <string name="label_copyright_value">© Wire Swiss GmbH</string>
```
Comment (vitorhugods) [inline app/src/main/res/values/strings.xml]: Maybe:

```suggestion
    <string translatable="false" name="label_copyright_value">© Wire Swiss GmbH</string>
```

**Review by vitorhugods — APPROVED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (alexandreferris) [reply app/src/main/res/values/strings.xml]: Yes! Good catch, also was not really sure if we should have it just in the `strings` file or something else

**Comment by AndroidBob** [comment]
Build [4732](https://10.10.124.134/job/AR-build-pipeline/4732/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.8.0(9671793)(9671793)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.8.0(9671793)(9671793)-staging-compat.apk)

**Pull request merged**